meteor-app/imports/api/members/members.js

```diff
@@ -1,33 +1,47 @@
-import { Mongo } from 'meteor/mongo';
+import { Mongo } from 'meteor/mongo'
+import { ValidatedMethod } from 'meteor/mdg:validated-method'
+import * as _ from 'lodash'
 
-const Members = new Mongo.Collection('members');
-
-Members.search = function(query){
-  console.log('searching method... ', query)
-  return Members.find({
-    firstname: { $regex: RegExp(query), $options: 'i' }
-  }, {
-    limit: 20
-  })
+class Member {
+  constructor(doc) {
+    _.extend(this, doc)
+    this.name = `${this.firstname} ${this.lastname}`
+  }
 }
 
+const Members = new Mongo.Collection('members', {
+  transform: (doc) => new Member(doc)
+})
+
+// 
+// Members.search = function(query){
+//   console.log('searching method... ', query)
+//   return Members.find({
+//     firstname: { $regex: RegExp(query), $options: 'i' }
+//   }, {
+//     limit: 20
+//   })
+// }
+
 Members.attachSchema(new SimpleSchema({
   firstname: {
     type: String,
     label: "Name",
     max: 128,
-    optional: false
+    defaultValue: "Firstname",
```

Comment:
This will leave a bunch of people in the database called "Firstname Lastname", which I don't think is ideal. 

As the name import is a one off, you can just comment out the attachment of the schema while you do it, either that or you trap the exceptions as they happen, and report on the ones that didn't make it into the dataset.